study/cyber-tutor/quiz/index.html

```diff
@@ -0,0 +1,121 @@
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <link rel="stylesheet" href="../../../css/bootstrap4.5.0.min.css">
+        <link rel="stylesheet" href="../../../codemirror/codemirror.css">
+        <style>
+            #example {
+                border: 1px solid rgba(0,0,0,.125);
+                border-radius: .25rem;
+                padding: 1rem;
+                width: 50%;
+                white-space: pre-wrap;
+            }
+
+            #notification {
+                position: absolute;
+                right: 5px;
+                width: 25%;
+                padding: .75rem 1.25rem;
+                margin-bottom: 1rem;
+                border: 1px solid transparent;
+                border-radius: .25rem;
+            }
+
+            #notification.success {
+                color: #155724;
+                background-color: #d4edda;
+                border-color: #c3e6cb;
+            }
+
+            #notification.failure {
+                color: #721c24;
+                background-color: #f8d7da;
+                border-color: #f5c6cb;
+            }
+
+            @media only screen and (max-width: 768px) {
+                #example {
+                    width: 90%;
+                }
+            }
+        </style>
+
+        <script src="https://kit.fontawesome.com/b3cde37b04.js" crossorigin="anonymous"></script>
+    </head>
+
+    <body>
+        <nav class="navbar navbar-expand-md bg-secondary navbar-dark mb-3">
+            <a class="navbar-brand" href="../../../">TeCanal Kids Corner</a>
+
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavbar">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <div class="collapse navbar-collapse" id="collapsibleNavbar">
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a class="nav-link" href="../../../play">
+                            <i class="fa fa-gamepad"></i>
+                            Play
+                        </a>
+                    </li>
+
+                    <li class="nav-item active">
+                        <i class="fa fa-paperclip"></i>
+                        Study
+                    </li>
+
+                    <li class="nav-item">
+                        <a class="nav-link" href="../../../read">
+                            <i class="fa fa-book"></i>
+                            Read
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+
+  <!--      <section>-->
+            <div id="notification" style="display: none;"></div>
+
+            <div class="container">
+                <div style="text-align: center;">
+                    <h1 style="margin-bottom: 0;">Quiz</h1>
+                </div>
+				<div id="quiz"></div>
+				<button id="submit">Check Answers</button>
```

Comment:
Make this a [Bootstrap Button](https://getbootstrap.com/docs/4.0/components/buttons/).